index.js

```diff
@@ -142,4 +142,4 @@ $('.day-second').click(function() {
     $('.day-first').removeClass('day-active');
     $('.day-second-events').show();
     $('.day-second').addClass('day-active');
-});
+});
```

Comment:
```suggestion
});

```
nit: newline end of file